plugin/pkg/admission/security/podsecuritypolicy/admission_test.go

```diff
@@ -1433,7 +1434,7 @@ func TestAdmitFSGroup(t *testing.T) {
 		// doesn't matter if we set it here or on the container, the
 		// admission controller uses DetermineEffectiveSC to get the defaulting
 		// behavior so it can validate what will be applied at runtime
-		pod.Spec.SecurityContext.FSGroup = groupIDPtr(group)
+		pod.Spec.SecurityContext.FSGroup = utilpointer.Int64Ptr(group)
```

Comment:
Could we change the type of the `group` to `int64`?

Reply:
Fixed